manager/src/main/java/com/redhat/service/bridge/manager/connectors/AbstractConnectorWorker.java

```diff
@@ -77,6 +77,12 @@ public void execute(ConnectorEntity currentEntity) {
         }
     }
 
+    private R externalServiceInteraction(ConnectorEntity connectorEntity) {
+        R serviceResponse = callExternalService(connectorEntity);
+        LOGGER.info("Worker {} successfully called service with response : {} on entity: {} ", workerName(), serviceResponse, connectorEntity);
```

Comment:
Are you sure logging the whole `connectorEntity` and `serviceResponse` is safe? As far as I remember there are cases where it sends service account as part of the payload
I see similar log in other places too so we should review them (maybe with a followup ticket)